### PR TITLE
Add section title to SearchFragment

### LINKED ISSUE #529: Add section title to SearchFragment
## Kind (Required)

- Other

 ## Overview (Required)

- Add section title to SearchFragment
- If you tell me email adress (or inviting to GitHub bio email adress), I will send invitation of Gallery.

image

 ## Links	

-https://gallery.io/projects/MCHbtQVoQ2HCZd9Pbpex4PVP/files/MCHtA7U1iMGr6-ZF5RXVf1KRWr8VHz3wbOk

### PR DESCRIPTION
**Pull request by a25max**
## Issue
- close #529 

## Overview (Required)
- create HeaderItem class. This class is show section title.
- use Header class because, show section title.

## Links
- https://github.com/lisawray/groupie

### Speaker
Before | After
:--: | :--:
 | 

### Sessions
Before | After
:--: | :--:
 |

### CODE REVIEW

File: frontendcomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2019/item/HeaderItem.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package io.github.droidkaigi.confsched2019.item
+
+import android.widget.TextView
+import com.xwray.groupie.Item
+import com.xwray.groupie.ViewHolder
+import io.github.droidkaigi.confsched2019.widget.component.R
+
+class HeaderItem(private val title: CharSequence) : Item<ViewHolder>() {
+
+    override fun getLayout(): Int = R.layout.section
```
Comment (panpanini) [inline frontendcomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2019/item/HeaderItem.kt]: the naming `section` sounds a little bit too generic, so I was confused at first.. do you think you could change it to something more specific? Maybe something like `view_section_header`?

File: frontendcomponent/androidcomponent/src/main/res/layout/section.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <TextView
+        android:id="@+id/section_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/gray1"
+        android:textSize="24sp"
+        android:text="@string/session_label"
+        android:layout_marginTop="22dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginBottom="27dp"
```
Comment (panpanini) [inline frontendcomponent/androidcomponent/src/main/res/layout/section.xml]: it looks like maybe this should be `28dp` 🤔 
From [Material.io](https://material.io/design/layout/spacing-methods.html#spacing)

> Padding refers to the space between UI elements. It’s an alternative spacing method to keylines, measured in increments of 8dp or 4dp.

So it looks better to be `28dp` I think!

screen shot 2019-01-22 at 22 37 08

File: frontendcomponent/androidcomponent/src/main/res/layout/section.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content">
+    <TextView
+        android:id="@+id/section_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/gray1"
+        android:textSize="24sp"
+        android:text="@string/session_label"
+        android:layout_marginTop="22dp"
```
Comment (panpanini) [inline frontendcomponent/androidcomponent/src/main/res/layout/section.xml]: Here as well I think should be `24dp` or `28dp` (it looks different between `Speaker` and `Sessions`, but maybe we can simplify and make them the same)

screen shot 2019-01-22 at 22 44 18

File: frontendcomponent/androidcomponent/src/main/res/layout/section.xml (hunk 2 above)
Comment (a25max) [reply frontendcomponent/androidcomponent/src/main/res/layout/section.xml]: sorry 🙇inspect then this.




So I was doing it at 27 dp. but, I will change 28dp. Thank you 😄

File: frontendcomponent/androidcomponent/src/main/res/layout/section.xml (hunk 3 above)
Comment (a25max) [reply frontendcomponent/androidcomponent/src/main/res/layout/section.xml]: I will change 28dp!

File: frontendcomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2019/item/HeaderItem.kt (hunk 1 above)
Comment (a25max) [reply frontendcomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2019/item/HeaderItem.kt]: Sorry 🙇 Be careful.
I will change `view_section_header`.

**Pull request merged**